starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,57 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <div class = "container">
+      <h1>Apple Pies</h1> 
+      <div class="paragraph"> <p>This was my grandmother's apple pie recipe. I have never seen another one
+        quite like it. It will always be my favorite and has won me several first place
+        prizes in local competitions. I hope it becomes one of your favorites as well! </p>  </div>
+        <div class="timeCook"> 
+          <img src="./images/cooking-info.png" alt="">
+        </div>
+   
```

Comment:
la mejor estructura seria
```
<div class="container">
<section>
<h1>Apple Pies</h1> 
      <div class="paragraph"> <p>This was my grandmother's apple pie recipe. I have never seen another one
        quite like it. It will always be my favorite and has won me several first place
        prizes in local competitions. I hope it becomes one of your favorites as well! </p>  </div>
        <div class="timeCook"> 
          <img src="./images/cooking-info.png" alt="">
        </div>
</section
</div>
```

Reply:
Pero bien indentado